lib/chingu/classic_game_object.rb

```diff
@@ -149,36 +154,36 @@ def height
     def size=(size)
       self.width, self.height = *size
     end
-    
+
     # Get objects width and height in an array
     def size
       [self.width, self.height]
     end
-      
 
     # Quick way of setting both factor_x and factor_y
     def factor=(factor)
       @factor = factor
       @factor_x = @factor_y = factor
     end
+
     alias scale= factor=
     alias scale factor
-          
+
     # Quick way of setting both center_x and center_y
     def center=(center)
       @center = center
       @center_x = @center_y = center
     end
-    
+
     # Get objects alpha-value (internally stored in @color.alpha)
     def alpha
       @color.alpha
     end
-    
+
     # Set objects alpha-value (internally stored in @color.alpha)
     # If out of range, set to closest working value. this makes fading simpler.
     def alpha=(value)
-      value = 0   if value < 0
+      value = 0 if value.negative?
       value = 255 if value > 255
       @color.alpha = value.to_i
```

Comment:
```suggestion
      @color.alpha = value.clamp(0, 255).to_i
```

Reply:
Actually never mind, Gosu::Color already clamps:
```suggestion
      @color.alpha = value
```